frontend/docs/introduction.md

```diff
@@ -18,8 +18,14 @@ project.
 # deno
 deno add jsr:@luca/cases
 
-# npm (use any of npx, yarn dlx, pnpm dlx, or bunx)
-npx jsr add @luca/cases
+# pnpm 10.9+
+pnpm add jsr:@luca/cases
+
+# yarn 4.9+
+yarn add @luca/cases:<todo>
```

Comment:
@arcanis I can't tell what the syntax here is meant to be that is equivalent to `pnpm add jsr:@luca/cases` or `deno add jsr:@luca/cases`. Is it `yarn add `@luca/cases@jsr:latest`? If that's it, that seems pretty long.